app/gitcoinbot/gitcoinbotActions.py

```diff
@@ -0,0 +1,133 @@
+"""
+Methods for interacting with the Github API as a Github App
+"""
+import datetime
+import json
+import re
+
+from django.conf import settings
+
+import jwt
+import requests
+from github.utils import post_issue_comment, post_issue_comment_reaction
+
+GITCOINBOT_APP_ID = settings.GITCOINBOT_APP_ID
+SECRET_KEYSTRING = settings.SECRET_KEYSTRING
+
```

Comment:
Can you add a second return between the module variables/imports here and all of the methods below?